scripts/ci/local_ci_enter_environment.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+#
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+
+set -xeuo pipefail
+MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck source=./_check_not_in_container.sh
+. "${MY_DIR}/_check_not_in_container.sh"
+
+export RUN_TESTS="false"
+export MOUNT_LOCAL_SOURCES="true"
+export PYTHON_VERSION=${PYTHON_VERSION:="3.6"}
+
+# shellcheck source=./ci_run_airflow_testing.sh
+"${MY_DIR}/ci_run_airflow_testing.sh"
```

Comment:
The content of this file seems identical(? very similar?) to `scripts/ci/ci_enter_environment.sh`

Reply:
Not really. It is intended to run locally, so - local sources are mounted, default python_version is set to 3.6 and verbosity is set to false (in CI we do not mount sources, take python version from the python on the path and verbosity is set to true).

export MOUNT_LOCAL_SOURCES="true"
export PYTHON_VERSION=${PYTHON_VERSION:="3.6"}
export VERBOSE=${VERBOSE:="false"}

However - we do not ever need the ci_enter_environment.sh in fact - it's only intended for local use. so I delete the ci_enter_environment.sh  :).